resolwe_bio/tools/star_sj_to_bed12.py

```diff
@@ -62,9 +62,9 @@
 
 # 6: strand =  0: '.' (undefined) , 1: '+', 2: '-
 conditions = [
-    sj_file.iloc[:, 4] == 0,
-    sj_file.iloc[:, 4] == 1,
-    sj_file.iloc[:, 4] == 2
+    sj_file.iloc[:, 3] == 0,
```

Comment:
Can you write a test that would detect this bug?

Reply:
The fix is quite straight-forward - the column index was assigned in 1-based coord. system, instead of -based. This is fixed now.

Writing a failing test would require to modify STAR SJ file. It would need to include motifs coded with numbers other than 0,1  or 2. This would indeed require to use totally different genome, reads and annotation test files. Basically generating new test files from scratch. I would merge this fix and leave test files to be fixed after September release...

Reply:
> Writing a failing test would require to modify STAR SJ file.

The passing test (the one we have now) should fail for wrong indexing.

Reply:
Do we have a test for this?

Reply:
As Jure described the indexing was correct. The problem is that test files include only splice junctions with intron motif 0: non-canonical; 1: GT/AG, 2: CT/AC and not 3: GC/AG, 4: CT/GC, 5: AT/AC, 6: GT/AT.  
So in  STAR SJ test file there are same numbers in 4th and 5th column, that is why the mistake happened.

Reply:
Thanks for the explanation.